lib/y-richtext.coffee

```diff
@@ -172,12 +172,20 @@ class YRichText extends BaseClass
 
       if delta.insert?
         insertHelper thisObj, position, delta.insert
-        from = thisObj._model.getContent("characters").ref(position)
-        to = thisObj._model.getContent("characters").ref(position+delta.insert.length)
-        thisObj._model.getContent("selections").select from, to, delta_selections
-        thisObj._model.getContent("selections").unselect from, to, delta_unselections
-
-        return position + delta.insert.length
+        length =
```

Comment:
`delta.insert` is defined, because you checked for `delta.insert?` I actually have the same concern as you, I want to sanitize the input so that we always get something that we expect. The ot-types are very likely to get changed a lot. Maybe you could throw an error, when something unexpected happens? (e.g. `delta.insert` is an object)